src/gemdat/path.py

```diff
@@ -205,6 +205,101 @@ def free_energy_graph(F: np.ndarray,
                                'dijkstra-exp']
 
 
+def calculate_path_difference(path_sites1: list, path_sites2: list) -> float:
+    """Calculate the difference between two paths. This difference is defined
+    as the percentage of sites that are not shared between the two paths.
+
+    Parameters
+    ----------
+    path_sites1 : list
+        List of sites defining the first path
+    path_sites2 : list
+        List of sites defining the second path
+
+    Returns
+    -------
+    difference : float
+        Difference between the two paths
+    """
+
+    # Find the shortest and longest paths
+    shortest_path = path_sites1 if len(path_sites1) <= len(
+        path_sites2) else path_sites2
+    longest_path = path_sites2 if shortest_path is path_sites1 else path_sites1
+
+    # Calculate the number of nodes shared between the shortest and longest paths
+    shared_nodes = 0
+    for node in shortest_path:
+        if node in longest_path:
+            shared_nodes += 1
+
+    return 1 - (shared_nodes / len(shortest_path))
+
+
+def multiple_paths(
+    F_graph: nx.Graph,
+    start: tuple,
+    end: tuple,
+    method: _PATHFINDING_METHODS = 'dijkstra',
+    Npaths: int = 3,
+    min_diff: float = 0.15,
```

Comment:
Best to force keyword arguments for such functions. This prevents errors when the api changes or if you mix up start/stop at some point.

Another point is to choose good 'opposites' as variable names. Either start<->stop, or begin<->end.

Finally, try to avoid capitals in variable names if you can.

```suggestion
    F_graph: nx.Graph,
    *,
    start: tuple,
    stop: tuple,
    method: _PATHFINDING_METHODS = 'dijkstra',
    n_paths: int = 3,
    min_diff: float = 0.15,
```

Reply:
Also can you update the types here for start/stop?

Reply:
done!